requirements.txt

```diff
@@ -1,4 +1,7 @@
+chainmap
```

Comment:
Is this necessary? [The future docs](http://python-future.org/quickstart.html?highlight=chainmap#standard-library-reorganization) seem to imply that normal Chainmap imports should just work.